bin/clean-tsv-metadata

```diff
@@ -2,7 +2,7 @@
 import pandas as pd
 import argparse
 
-def clean_metadata_file(file_name, n_cols, header):
+def clean_metadata_file(file_name, n_cols, header, sort_col):
```

Comment:
I think changes in this file accidentally got pushed to this commit that adds a new workflow; can you split them out?